packages/jest-create-cache-key-function/src/__tests__/index.test.ts

```diff
@@ -44,3 +47,27 @@ test('creation of a cache key', () => {
   expect(hashA).not.toEqual(hashB);
   expect(hashA).not.toEqual(hashC);
 });
+
+test('creation of a cache key on win32', () => {
+  process.platform = 'win32';
+  const createCacheKeyFunction = interopRequireDefault(
+    require('../index'),
+  ).default;
+  const createCacheKey = createCacheKeyFunction([], ['value']);
+  const hashA = createCacheKey('test', 'test.js', null, {
+    config: {},
+    instrument: false,
+  });
+  const hashB = createCacheKey('test code;', 'test.js', null, {
+    config: {},
+    instrument: false,
+  });
+  const hashC = createCacheKey('test', 'test.js', null, {
+    config: {},
+    instrument: true,
+  });
+
+  expect(hashA).toHaveLength(16);
```

Comment:
Thanks. I think you can simplify the test and just check the length. The rest is tested above.

Would be great to add another test for custom length of the key.